### ISSUE TITLE
Change (Omitted Text) to footnotes

### ISSUE THREAD

**Issue by DavidHaslam** [lemtom/coverdale#7]
There are **14** instances of `(Omitted Text)`:
```
        <verse osisID="Exod.38.15"> (Omitted Text)
        <verse osisID="Lev.15.23"> (Omitted Text)
        <verse osisID="Num.7.64"> (Omitted Text)
        <verse osisID="Deut.14.20"> (Omitted Text)
        <verse osisID="Josh.15.52"> (Omitted Text)
        <verse osisID="Josh.15.53"> (Omitted Text)
        <verse osisID="Josh.15.54"> (Omitted Text)
        <verse osisID="1Kgs.6.31"> (Omitted Text)
        <verse osisID="Neh.13.27"> (Omitted Text)
        <verse osisID="Ps.136.26"> (Omitted Text)
        <verse osisID="Mark.6.46"> (Omitted Text)
        <verse osisID="Mark.11.26"> (Omitted Text)
        <verse osisID="Luke.17.36"> (Omitted Text)
        <verse osisID="Rev.21.26"> (Omitted Text)
```

As this is not canonical content, it would be better placed inside a **note** element.

**Comment by DavidHaslam** [comment]
Please also add an explanation for `(Omitted Text)` in the [README](https://github.com/lemtom/coverdale/blob/master/README.md) file.

**Comment by DavidHaslam** [comment]
Fixed in merge request #8
